packages/preset-uno/src/rules/animation.ts

```diff
@@ -14,4 +15,42 @@ export const animations: Rule[] = [
     return `${keyframes[name]}\n${constructCSS({ animation: `${name} 1s linear infinite` })}`
   }],
   ['animate-none', { animation: 'none' }],
+  [/^animate(?:-duration)?-((\d+)(?:(s|ms)?))$/, ([, d]) => {
+    return {
+      'animation-duration': h.bracket.time(d.replace(/-duration/, '')),
+    }
+  }],
+  [/^animate-delay-((\d+)(?:(s|ms)?))$/, ([, d]) => {
+    return {
+      'animation-delay': h.bracket.time(d),
+    }
+  }],
+  [
+    /^animate-(?:fill-)?mode-(none|forwards|backwards|both|inherit|initial|revert|unset)$/, ([, d]) => ({
+      'animation-fill-mode': d,
+    }),
+  ],
+  [
+    /^animate-(?:direction)?-(normal|reverse|alternate|alternate-reverse|inherit|initial|revert|unset)$/, ([, d]) => ({
+      'animation-direction': d,
```

Comment:
should include second `-` inside `direction` captring group and remove  `direction-` from value

Reply:
forgot about remove value, just the second `-`